### ISSUE TITLE
Please allow manual toggle of single-file mode

### ISSUE THREAD

**Issue by elliotchance** [Reviewable/Reviewable#261]
There is a magic threshold where it decides the diff is too big and goes into single file mode, with the option to go back to multi-file mode.

We need the opposite of this. Sometimes the diff is big enough that the browser really lags and I'd like to be able to manually change it into single file review mode.

**Comment by pkaminski** [comment]
Yep, this is in the pipeline.  In the meantime, you can {ctrl,cmd,shift}-click on a file to open a tab in a special single-file mode.

**Comment by kballard** [comment]
The threshold is also too high I think. I'm dealing with a review right now that is more than large enough to cause Reviewable to get really slow and use up a lot of memory, but for some reason Reviewable doesn't think it's worth switching into single-file mode.

**Comment by al-the-x** [comment]
I'd also _always_ prefer single-file mode, particularly when there are a _ton_ of comments on a small number of files. I have another developer on my team who just prefers to work that way, so having an explicit toggle for single-file mode would really help. The "{ctrl,cmd,shift}-click" trick is handy, though.
